src/batou/tests/test_deploy.py

```diff
@@ -31,7 +31,7 @@ def test_main_with_errors(capsys):
     assert out == Ellipsis(
         """\
 batou/2... (cpython 3...)
-================================== Preparing ===================================
+... Preparing ...
```

Comment:
It's not quite clear why this changes when changing the encryption?

Reply:
This is not directly related to the encryption but was required so I can run the test suite on my machine to test all other changes. The specific commit is d4caccaf4239dc1d7909859a2fb1f63353c62a3b. I can move these out to a separate PR now if you want.